### ISSUE TITLE
assumego is gone

### ISSUE THREAD

**Issue by mnichols** [dex4er/asdf-granted#1]
Plugin fails to complete installation since `assumego` is no longer a thing
https://github.com/common-fate/granted/pull/404

**Comment by dex4er** [comment]
@mnichols I think it is a bug in a tarball for Mac:

```
 dex4er > ~ > % grep assumego /Users/dex4er/.asdf/installs/granted/0.11.1/bin/assume
#GRANTED_FLAG - what assumego told the shell to do
#GRANTED_n - the data from assumego
GRANTED_OUTPUT=$(assumego "$@")
```

Ok I'll create a symlink then.

**Comment by dex4er** [comment]
Fixed in 24f20f7f727778572ebbc2683bcfd78cf3289e01
